docs/dev.md

```diff
@@ -40,13 +31,15 @@ curl -X PUT -d s3kretKey http://localhost:8080/api/admin/settings/:BlockedApiKey
 curl -X PUT -d unblock-key http://localhost:8080/api/admin/settings/:BlockedApiPolicy
 ```
 
-### dd-ingest-flow
+### dd-ingest-flow + dd-validate-dans-bag/application
+
+In separate terminal tabs:
```

Comment:
```suggestion
Open both project in separate terminal tabs do the following for each:
```

Reply:
Btw note that this all a bit redundant anyway, as the start-env.sh and start-service.sh scripts are documented in dans-dev-tools.